modules/core/src/main/java/org/apache/ignite/internal/util/ByteUtils.java

```diff
@@ -344,4 +345,16 @@ public static List<byte[]> toByteArrayList(List<ByteBuffer> byteBufferList) {
 
         return unmodifiableList(result);
     }
+
+    /**
+     * Converts a UUID to bytes.
+     */
+    public static byte[] uuidToBytes(UUID uuid) {
```

Comment:
Because we need to use `ByteUtils#toBytes()` as never as possible :) Custom serialization is more compact and faster. Also, in this case, it's really easy to write.

Reply:
let's add a test then in `org.apache.ignite.internal.util.ByteUtilsTest`

Reply:
Added a test